.kitchen.yml

```diff
@@ -18,11 +18,11 @@ platforms:
   - name: windows-2012r2
     driver_config:
       box: chef/windows-server-2012r2-standard
-  - name: windows-2012r2-chef-12.6
-    driver_config:
-      box: chef/windows-server-2012r2-standard
-    provisioner:
-      require_chef_omnibus: 12.6.0
+  # - name: windows-2012r2-chef-12.6
```

Comment:
I wasn't sure if that was still necessary for the kitchen testing. I can put it back in though if we still need to test the 12.6 client specifically? I defer to your judgement.